sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_aws_sdk.ipynb

```diff
@@ -0,0 +1,889 @@
+    "    status = sagemaker_client.describe_processing_job(ProcessingJobName=processing_job_name)\n",
+    "    print(status[\"ProcessingJobStatus\"])\n",
+    "    time.sleep(60)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Viewing the Bias Report\n",
+    "In Studio, you can view the results under the experiments tab.\n",
+    "\n",
+    "<img src=\"./recordings/bias_report.gif\">\n",
+    "\n",
+    "Each bias metric has detailed explanations with examples that you can explore.\n",
+    "\n",
+    "<img src=\"./recordings/bias_detail.gif\">\n",
+    "\n",
+    "You could also summarize the results in a handy table!\n",
+    "\n",
+    "<img src=\"./recordings/bias_report_chart.gif\">\n",
+    "\n",
+    "If you're not a Studio user yet you can access the complete analysis report at the following S3 bucket.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "analysis_result_path"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Explaining Predictions\n",
+    "There are expanding business needs and legislative regulations that require explanations of _why_ a model made the decision it did. SageMaker Clarify uses SHAP to explain the contribution that each input feature makes to the final decision."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Configure a SageMaker Clarify Processing Job Container's Input and Output Parameters "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "explainability_analysis_config_path = \"s3://{}/{}/explainability_analysis_config.json\".format(\n",
+    "    bucket, prefix\n",
+    ")\n",
+    "analysis_result_path = \"s3://{}/{}/output\".format(bucket, prefix)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Configure Analysis Config\n",
+    "For our example use case we will be using the following analysis config:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!echo\n",
+    "!cat explainability_analysis_config.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`explainability_analysis_config.json` here contains configuration values for detecting bias using a clarify container:\n",
+    "* `dataset_type` specify the format of your dataset, for thsi example as we are using csv dataset this will be `text/csv`\n",
+    "* `headers` is the list of column names in the dataset\n",
+    "* `label` specified the target attribute for the model to be used for bias metrics. Specified as a column name, or as index if the dataset format is CSV\n",
```

Comment:
Major comment: the description may cause confusion: if it is for bias analysis how come it is included in explainability analysis config? To avoid confusion we can say,

* `label` specifies  the ground truth label, in this example the "Target" column. The SageMaker Clarify job will drop the column and uses the remaining feature columns for explainability analysis.